tests/test_subtractTask.py

```diff
@@ -143,10 +140,7 @@ def test_clear_template_mask(self):
         diffimMask = output.difference.mask
         for maskPlane, scienceSum in scienceMaskCheck.items():
             diffimSum = np.sum(diffimMask.array & mask.getPlaneBitMask(maskPlane) > 0)
```

Comment:
Highlighting here, but the line above about "Except the "DETECTED" planes should have been cleared" should be deleted. Please check for other places in the tests that make that claim about the mask, since it's now false.